core/src/main/java/org/apache/calcite/rel/externalize/RelJson.java

```diff
@@ -622,6 +622,7 @@ public Object toJson(RexNode node) {
         }
         map.put("operands", list);
         switch (node.getKind()) {
+        case MINUS:
```

Comment:
We should change the Jira issue to reflect that we are only changing the behavior of `MINUS`.